config/helmchart/templates/manager.yaml

```diff
@@ -17,13 +17,29 @@ spec:
     {{- end }}
       labels:
         {{- include "egressip-ipam-operator.selectorLabels" . | nindent 8 }}
+        control-plane: controller-manager
```

Comment:
Do we want to align with `operator: egressip-ipam-operator`?

Reply:
fixed